spec/database_helpers.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'pg'
+
+def persisted_data(id:,table:)
+  connection = PG.connect(dbname: 'bookmark_manager_test')
```

Comment:
I know you changed it already! It should be chitter_test and not bookmark_manager_test.